packages/typespec-client-generator-core/test/decorators/protocol-api.test.ts

```diff
@@ -49,3 +49,60 @@ describe("@protocolAPI", () => {
     await protocolAPITestHelper(runner, false, false);
   });
 });
+
+describe("@protocolAPI on interface", () => {
+  it("applies protocolAPI false to all operations in interface", async () => {
+    const testCode = `
+      @service
+      namespace MyService {
+        @protocolAPI(false)
```

Comment:
Can I set @protocolAPI and @convenientAPI at the same operation?  Just curious the behavior once we set both of them as false for an interface.